components/board/genericlinux/board.go

```diff
@@ -407,24 +407,23 @@ func (b *Board) AnalogByName(name string) (board.Analog, error) {
 }
 
 // DigitalInterruptByName returns the interrupt by the given name if it exists.
-func (b *Board) DigitalInterruptByName(name string) (board.DigitalInterrupt, bool) {
+func (b *Board) DigitalInterruptByName(name string) (board.DigitalInterrupt, error) {
 	b.mu.Lock()
 	defer b.mu.Unlock()
 
 	interrupt, ok := b.interrupts[name]
 	if ok {
-		return interrupt.interrupt, true
+		return interrupt.interrupt, nil
 	}
 
 	// Otherwise, the name is not something we recognize yet. If it appears to be a GPIO pin, we'll
 	// remove its GPIO capabilities and turn it into a digital interrupt.
 	gpio, ok := b.gpios[name]
 	if !ok {
-		return nil, false
+		return nil, errors.Errorf("cant find GPIO (%s)", name)
```

Comment:
use `fmt.Errorf` instead.

Reply:
the last file you used Errorsf in didn't have the fmt package pulled in. (the pi driver)
genericlinux does, so let's not use the public archive errors package.